source/install/prod-windows-2012.rst

```diff
@@ -0,0 +1,419 @@
+
+Configure SSL
+^^^^^^^^^^^^^
+
+.. note:: SSL communication requires that the web server have a well-formed and trusted certificate.  A common
+   freely-available SSL encryption and certificate managemet is Let's Encrypt; however, this service does
+   not formally support the Windows Operating system. A number of third-parties have created clients to support this, and
+   you are free to try out any of them.  This section assumes that you have taken the necessary steps
+   to obtain a web-server certificate that will be trusted by your users.
+
+29. Within the IIS Manager, select the server node in the left-hand connections pane.
+
+30. Double-click the **Server Certificates** option.
+
+31. Select **Import...** from the list of actions on the right-hand-side.
+
+32. Press the ... button to locate your PFX formatted certificate.
+
+33. Enter the password to the certificate file.
+
+34. Select the **Web Hosting** certificate store, and press OK.
+
+35. In the left-hand navigation tree, expand the server node, expand **Sites**, and select **Default Web Site**.
+
+36. In the right-hand-side **Actions** pane, select **Bindings...**.
+
+37. Press **Add**
+
+38. In the **Add Site Binding** dialog, Set the type to **https** and set the **SSL Certificate** to the certificate loaded previously.  Press OK.
+
+Redirect HTTP to HTTPS
+^^^^^^^^^^^^^^^^^^^^^^
+
+39. In the left-hand navigation tree, expand the server node, expand **Sites** and select **Default Web Site**.
+
+40. Double-click to open the Url Rewrite feature.
+
+41. In the actions-pane (far-right), select **Add Rule(s)..**
+
+42. Under the **Inbound Rules** section, select **Blank rule** and press OK.
+
+43. Populate the fields in the rule to match the screenshot below
+
+    .. image:: ../images/windows_6_http_to_https_redirect.png   
+
+44. Press **Apply** and then press **Back to Rules**.
+
+45. Ensure that the **HTTP to HTTPS Redirect** rule is at the top of the list of inbound rules.  If nceessary, you can select a rule and use the **Move Up** and **Move Down** actions to reorganize.
+
+46. On a client workstation, open your favorite browser and navigate to `http://mattermost.example.com` and confirm that you are redirected to `https://mattermost.example.com`.
+
+Finish Mattermost Server Setup
+------------------------------
+
+1. Navigate to ``https://mattermost.example.com`` and create a user and team.
+
+2. The first user in the system is automatically granted the
+   ``system_admin`` role, which gives you access to the System Console.
+
+3. From the ``town-square`` channel click the dropdown next to your team name and choose the
+   ``System Console`` option
+   
+4. Update **General** > **Configuration** settings to properly configure your reverse proxy by entering `https://mattermost.example.com` as the **Site URL**
+
+   ..alert:: Failure to properly set the Site URL properly __will__ result in unexpected behavior.  Do not include the trailing '/' on the URL.
```

Comment:
I think the formatting for `..alert::` isn't expected:

![image](https://cloud.githubusercontent.com/assets/13119842/20434297/3973b010-ad75-11e6-84a9-a1911ed0f2d2.png)